packages/app/src/specs/SpecsListBanners.cy.tsx

```diff
@@ -153,7 +154,20 @@ describe('<SpecsListBanners />', { viewportHeight: 260, defaultCommandTimeout: 1
           }
         })
 
-        cy.get(`[data-cy="${bannerTestId}"]`).should('be.visible')
+        cy.get(`[data-cy="${bannerTestId}-banner"]`)
+        .should('be.visible')
+
+        // The ct title has dynamic content and seems complicated to set here, so ignoring
+        let ctBannerTitle = defaultMessages.specPage.banners.componentTesting.title
+
+        if (ctBannerTitle) {
+          defaultMessages.specPage.banners.componentTesting.title = ctBannerTitle.replace('{0}', 'React')
+        }
+
+        cy.wrap(Object.entries(defaultMessages.specPage.banners[camelCase(bannerTestId)])).each((entry) => {
```

Comment:
I added this assertion because after my updates, the rendering of the banner content was broken, but I only detected this through Percy. We should have a test that asserts on the content for the future.